Examples/src/main/kotlin/com/mapbox/vision/examples/POIView.kt

```diff
@@ -0,0 +1,167 @@
+package com.mapbox.vision.examples
+
+import android.content.Context
+import android.graphics.Bitmap
+import android.graphics.Canvas
+import android.graphics.Paint
+import android.graphics.Rect
+import android.os.Handler
+import android.util.AttributeSet
+import android.view.View
+import com.mapbox.mapboxsdk.geometry.LatLng
+import com.mapbox.vision.VisionReplayManager
+import com.mapbox.vision.examples.POIActivityKt.POI
+import com.mapbox.vision.mobile.core.models.frame.ImageSize
+import com.mapbox.vision.mobile.core.models.position.GeoCoordinate
+import com.mapbox.vision.mobile.core.models.position.GeoLocation
+import com.mapbox.vision.mobile.core.models.world.WorldCoordinate
+import kotlin.math.max
+import kotlin.math.min
+
+class POIView
+@JvmOverloads
+constructor(
+    context: Context,
+    attrs: AttributeSet? = null
+) : View(context, attrs) {
+
+    companion object {
+
+        // POI will start to appear at this distance, starting with transparent and appearing gradually
+        private const val DRAW_LABEL_MIN_DISTANCE_METERS = 250
```

Comment:
I'd put bigger value here - I guess it would be nicer.

Reply:
increased to 400, seems better